libflux/flux/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "flux"
-version = "0.5.1"
-authors = ["jlapacik <joshua.lapacik@pacbell.net>"]
+version = "0.154.0"
```

Comment:
I'd like to add a step to automatically bump the version in the Cargo file too. But the `cargo-bump` utility we used doesn't yet support workspace based projects. See https://github.com/wraithan/cargo-bump/blob/master/src/config.rs#L102 

We can add that behavior of the release later.

Reply:
Yeah, this is one of those things that has always bothered me.

There are a couple of cargo tools that don't with workspaces that I use regularly. It makes me sad.

Reply:
Submitted PR to cargo-bump for workspace support https://github.com/wraithan/cargo-bump/pull/34